spec/posix/1.0.md

```diff
@@ -7,26 +7,21 @@ The canonical URL for this document is <https://vt6.io/std/posix/1.0/>.
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
+Unless stated otherwise, this document implies all definitions of terms made by [`vt6/foundation`](https://vt6.io/std/foundation/) and other referenced modules.
+
 ## 1. Scope
 
-Operating systems that implement the [POSIX.1 specification](http://pubs.opengroup.org/onlinepubs/9699919799/) are admissible environments for VT6 clients and servers.
+Operating systems that implement the [POSIX.1 specification](http://pubs.opengroup.org/onlinepubs/9699919799/) are admissible environments for VT6 clients and terminals.
 This module defines how various platform-specific mechanisms throughout VT6 work on such operating systems.
 
-## 2. Definitions for the `vt6/core1.0` module
-
-Standard input and output for each VT6 client are the file descriptors 0 and 1 of that process, respectively.
-
-POSIX allows processes to be started while a VT6 server is not present.
-Therefore, VT6 clients MUST use the following method to determine whether a VT6 server is present.
+## 2. Definitions for `vt6/foundation`
```

Comment:
May this be a section `foundation` should require module specifications to always include?

Reply:
No. This structure is specific to platform integration modules.